habitat-lab/habitat/datasets/rearrange/configs/rearrange_floorplanner.yaml

```diff
@@ -1,1329 +1,468 @@
+- name: test_unseen_candle_holder
+  included_substrings:
+  - B075HXHKTZ
+  excluded_substrings: []
+- name: test_unseen_canister
+  included_substrings:
+  - B07MF1SFKD
+  - B07M6PJ66Q
+  excluded_substrings: []
+- name: test_unseen_dishtowel
+  included_substrings:
+  - Tag_Dishtowel_18_x_26
+  - Cole_Hardware_Dishtowel_Red
+  - Cole_Hardware_Dishtowel_Multicolors
+  - Cole_Hardware_Dishtowel_Stripe
+  - Envision_Home_Dish_Drying_Mat_Red_6_x_18
+  excluded_substrings: []
+- name: test_unseen_hat
+  included_substrings:
+  - DPC_tropical_Trends_Hat
+  excluded_substrings: []
+- name: test_unseen_medicine_bottle
+  included_substrings:
+  - QAbsorb_CoQ10_53iUqjWjW3O
+  - Folic_Acid
+  - CoQ10_wSSVoxVppVD
+  - Borage_GLA240Gamma_Tocopherol
+  - Quercetin_500
+  - Prostate_Optimizer
+  - Theanine
+  - Pet_Dophilus_powder
+  - Perricone_MD_Health_Weight_Management_Supplements
+  - Beta_Glucan
+  excluded_substrings: []
+- name: test_unseen_mouse_pad
+  included_substrings:
+  - B003QTD4Y6
+  excluded_substrings: []
+- name: test_unseen_pencil_case
+  included_substrings:
+  - Big_Dot_Pink_Pencil_Case
+  - Olive_Kids_Robots_Pencil_Case
+  - Olive_Kids_Paisley_Pencil_Case
+  excluded_substrings: []
+- name: test_unseen_plate
+  included_substrings:
+  - Threshold_Dinner_Plate_Square_Rim_White_Porcelain
+  excluded_substrings: []
+- name: test_unseen_scissors
+  included_substrings:
+  - Diamond_Visions_Scissors_Red
+  excluded_substrings: []
+- name: test_unseen_screwdriver
+  included_substrings:
+  - Craftsman_Grip_Screwdriver_Phillips_Cushion
+  excluded_substrings: []
+- name: test_unseen_shoe
+  included_substrings:
+  - TOP_TEN_HI_60KlbRbdoJA
+  - Sperry_TopSider_pSUFPWQXPp3
+  - Santa_Cruz_Mens_YmsMDkFf11Z
+  - D_ROSE_45
+  - PHEEHAN_RUN
+  - PureConnect_2_Color_AnthrcteKnckoutPnkGrnGecko_Size_50
+  - F10_TRX_FG_ssscuo9tGxb
+  - ASICS_GELTour_Lyte_WhiteOrchidSilver
+  - ASICS_GELAce_Pro_Pearl_WhitePink
+  - Santa_Cruz_Mens_vnbiTDDt5xH
+  - Copperhead_Snake_Tieks_Brown_Snake_Print_Ballet_Flats
+  - REEF_BRAIDED_CUSHION
+  - ASICS_GEL1140V_WhiteBlackSilver
+  - Ravenna_4_Color_WhtOlyBluBlkShkOrngSlvRdO_Size_70
+  - ASICS_GEL1140V_WhiteRoyalSilver
+  - REEF_BANTU
+  - Cascadia_8_Color_AquariusHibscsBearingSeaBlk_Size_50
+  - PureCadence_2_Color_HiRskRedNghtlfeSlvrBlckWht_Size_70
+  - ASICS_HyperRocketgirl_SP_5_WhiteMalibu_BlueBlack
+  - ENFR_MID_ENFORCER
+  - PureCadence_2_Color_TleBluLmePnchSlvMoodIndgWh_Size_50_EEzAfcBfHHO
+  - FYW_DIVISION
+  - Azure_Snake_Tieks_Leather_Snake_Print_Ballet_Flats
+  - Colton_Wntr_Chukka_y4jO0I8JQFW
+  - Crazy_Shadow_2_oW4Jd10HFFr
+  - ASICS_GELLinksmaster_WhiteRasberryGunmetal
+  - Adrenaline_GTS_13_Color_DrkDenimWhtBachlorBttnSlvr_Size_50_yfK40TNjq0V
+  - PureConnect_2_Color_BlckBrllntBluNghtlfeAnthrct_Size_70
+  - TROCHILUS_BOOST
+  - Chelsea_BlkHeelPMP_DwxLtZNxLZZ
+  - ASICS_GELLinksmaster_WhiteCoffeeSand
+  - D_ROSE_ENGLEWOOD_II
+  - Court_Attitude
+  excluded_substrings: []
+- name: test_unseen_spatula
```

Comment:
To confirm, these names are matching our decided category names correct?

Reply:
Yes, I did a manual clean-up of category names for a subset of categories before creating this split. But we are yet to work on finalizing the category names for the complete set of objects.

Reply:
Here's the [sheet](https://docs.google.com/spreadsheets/d/1EnAVC7YMdjA2s3xXKyabjNJdoB-b4Bd7mZTpK5hAMTE/edit?usp=sharing) containing the cleaned category names.